cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/namespace/AbstractNamespaceResourceDeleter.java

```diff
@@ -93,8 +90,6 @@ public void deleteResources(NamespaceMeta namespaceMeta) throws Exception {
     dashboardStore.delete(namespaceId.getNamespace());
     // Delete all applications
     applicationLifecycleService.removeAll(namespaceId);
-    // Delete all the schedules
-    scheduler.deleteAllSchedules(namespaceId);
```

Comment:
the program schedules are deleted as a part of the previous line:
`applicationLifecycleService.removeAll(namespaceId);`
In it, it deletes each application, which internally removes the schedules for that application.